hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -369,6 +369,10 @@ public final class OzoneConfigKeys {
       "ozone.fs.iterate.batch-size";
   public static final int OZONE_FS_ITERATE_BATCH_SIZE_DEFAULT = 100;
 
+  public static final String OZONE_FS_LIST_STATUS_BATCH_SIZE =
+          "ozone.fs.list.status.batch-size";
```

Comment:
```suggestion
          "ozone.fs.list.status.pagination-count";
```

Reply:
I think pagination is a better term, also other places in this PR we should convert batch to pagination.